doc/miniasync_dml_vdm.3.md.in

```diff
@@ -0,0 +1,62 @@
+---
+layout: manual
+Content-Style: 'text/css'
+title: _MP(MINIASYNC_DML_VDM, 3)
+collection: miniasync_dml
+header: MINIASYNC_DML_VDM
+secondary_title: miniasync_dml
+...
+
+[comment]: <> (SPDX-License-Identifier: BSD-3-Clause)
+[comment]: <> (Copyright 2021, Intel Corporation)
+
+[comment]: <> (miniasync_dml_vdm.3 -- man page for miniasync-dml vdm API)
+
+[NAME](#name)<br />
+[SYNOPSIS](#synopsis)<br />
+[DESCRIPTION](#description)<br />
+[ERRORS](#errors)<br />
+[SEE ALSO](#see-also)<br />
+
+
+# NAME #
+
+**miniasync_dml_vdm** - Virtual data mover API for miniasync-dml library
+
+
+# SYNOPSIS #
+
+```c
+#include <libminiasync.h>
+#include <libminiasync-dml.h>
+
+struct vdm_descriptor *vdm_descriptor_dml(void);
+struct vdm_descriptor *vdm_descriptor_dml_async(void);
+```
+
+For general description of miniasync see **miniasync**(7).
+
+
+# DESCRIPTION #
+
+**vdm_descriptor_dml**() and **vdm_descriptor_dml_async**() are **miniasync**(7)
+virtual data mover implementations of Data Mover Library (DML).
+
+`struct vdm_descriptor *vdm_descriptor_dml(void);`
+
+:	Data Mover Library implementation of synchronous virtual data mover.
+
+`struct vdm_descriptor *vdm_descriptor_dml_async(void);`
+
+:	Data Mover Library implementation of asynchronous virtual data mover.
+
```

Comment:
Can you elaborate a little bit what's the difference between the two and what is the expected behavior?

Reply:
DML provides blocking and nonblocking functions for job execution (dml_execute_job/dml_submit_job). vdm_descriptor_dml provides blocking behavior, whereas vdm_descriptor_dml_async provides nonblocking one.
I can remove the synchronous implementation if you want.
It was easier for me to write sync implementation first, so I could understand the miniasync code better.

Reply:
I meant elaborate in the man page :)
But yes, I think it would be easier to just remove the sync one.

Reply:
My bad :)
It is more informative now.